docs/develop/extended-overrides.md

```diff
@@ -8,7 +8,7 @@ ms.localizationpriority: medium
 
 # Work with Extended Overrides of the manifest
 
-Some extensibility features of Office Add-ins are configured with JSON files that are hosted on your server, instead of with the add-in's XML manifest.
+When your add-in uses the XML-formattted add-in only manifest, some extensibility features of Office Add-ins are configured with JSON files that are hosted on your server, instead of with the add-in's manifest.
```

Comment:
```suggestion
When your add-in uses the XML-formatted add-in only manifest, some extensibility features of Office Add-ins are configured with JSON files that are hosted on your server, instead of with the add-in's manifest.
```